docs/getting_started/python.md

```diff
@@ -0,0 +1,87 @@
+from django.contrib.auth.models import User                                     
+from django.urls import reverse                                                 
+                                                                                
+                                                                                
+class Task(models.Model):                                                       
+    user = models.ForeignKey(User, models.CASCADE)                                 
+    descr = models.CharField(max_length=12)                                        
+    due_date = models.DateField()                                                  
+    done = models.BooleanField(default=False)                                      
+                                                                                   
+    HIGH = 'high'                                                                  
+    LOW = 'low'                                                                    
+    PRIO = (                                                                       
+        (HIGH, HIGH),                                                              
+        (LOW, LOW),                                                                
+    )                                                                              
+    prio = models.CharField(max_length=4, choices=PRIO, default=LOW)               
+                                                                                   
+    def get_absolute_url(self):                                                    
+        return reverse('task_detail', args=(self.id,)) 
+```
+
+Now we need to create a migration for the above model and apply it:
+
+```
+cd ~/todo
+python manage.py makemigrations todo.todo
+python manage.py migrate
+```
+
+We are almost there. We need to add some urls and views to handle the urls.
+
+Open `~/todo/todo/urls.py` and add `path('', include('todo.urls'))` to your existing `urlpatterns`
+
+Your `~/todo/todo/todo/urls.py` should look like this:
+
+```
+from django.urls import path                                                    
+from . import views                                                             
+                                                                                
+urlpatterns = [                                                                 
+    path('', views.HomeView.as_view(), name='home'),                             
+    path('task/create/', views.TaskCreateView.as_view(), name='task_create'),    
+    path('task/list/done/', views.TaskListDoneView.as_view(), name='task_list_done'),
+    path('task/<int:id>/', views.TaskDetailView.as_view(), name='task_detail'), 
+]
+```
+
+Last step is to create the `views.py` file. 
+
+This will be a rather larger file and the content is omitted here, but it can be found at this [link](https://raw.githubusercontent.com/mvnm/django-onem-todo/master/todo/todo/views.py)
```

Comment:
I suggest to include the content of the file in this page as it's easier to read (syntax highlighting).

Reply:
I will need to rewrite the whole tutorial, so this bit wont be here anymore